### ISSUE TITLE
Skip conversion step for operations like isEmpty

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-simplify#157]
```elm
Dict.fromList >> Dict.isEmpty
--> List.isEmpty

Set.fromList >> Set.isEmpty
--> List.isEmpty

✅
Dict.toList >> List.isEmpty
--> Dict.isEmpty

✅
Set.toList >> List.isEmpty
--> Set.isEmpty

✅
Dict.toList >> List.length
--> Dict.size

✅
Set.toList >> List.length
--> Set.size

-- Note: This one doesn't work
Dict.fromList >> Dict.size
--> List.length
```

I'm sure we can find more operations like this for `Dict`/`Set`/`List`. And maybe there are similar things we can do for `Maybe`/`Result`.

**Comment by morteako** [comment]
@jfmengels  Some of these got implemented in : https://github.com/jfmengels/elm-review-simplify/pull/293

The two missing (not including the one that doesn't work 😅 )
```
Dict.fromList >> Dict.isEmpty
--> List.isEmpty

Set.fromList >> Set.isEmpty
--> List.isEmpty
```

**Comment by jfmengels** [comment]
Implemented in #293 and #303 .
